gym/monitoring/video_recorder.py

```diff
@@ -281,7 +281,10 @@ def capture_frame(self, frame):
         if frame.dtype != np.uint8:
             raise error.InvalidFrame("Your frame has data type {}, but we require uint8 (i.e. RGB values from 0-255).".format(frame.dtype))
 
-        self.proc.stdin.write(frame.tobytes())
+        if np.__version__ >= '1.9.0':
```

Comment:
Think we need to do something more sophisticated here:

```
>>> '1.10.0' >= '1.9.0'
False
```

Try using Disutils version parsing, like we do here: https://github.com/openai/mujoco-py/blob/master/mujoco_py/config.py#L45?